.azure/scripts/setup-helm.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -x
 
+HELM3_VERSION=${TEST_HELM3_VERSION:-'v3.12.0'}
```

Comment:
I'm not sure I follow the logic. Why not just rename all uses of `TEST_HELM3_VERSION` to `HELM3_VERSION` instead of using it `TEST_HELM3_VERSION` in the CI pipelines but renaming it here? Does it have some value? Or is it just an unnecessary layer of obfuscation?

Reply:
yeah I can change it it makes sense